octo_client/client.py

```diff
@@ -56,7 +58,10 @@ def _make_request(self, http_method: Callable, path: str, json=None, params=None
         )
         self._raise_for_status(response.status_code, response.content)
         try:
-            response_json = response.json()
+            if len(response.text) > DATADOG_LOG_SIZE_LIMIT:
+                response_json = 'TRUNCATED'
```

Comment:
Maybe we can return json instead of string ;)

Reply:
doesn't matter tbh